webapp-django/crashstats/crashstats/templates/crashstats/home.html

```diff
@@ -61,7 +61,7 @@ <h2 id="homepage-heading">{{ product }} {% if version %}{{ version }}{% endif %}
                             </a>
                         </li>
                         <li>
-                            <a href="{{ url('topcrashers:topcrashers') }}?product={{ product }}&amp;versions={{ version }}&amp;crash_type=plugin&amp;days={{days}}">
+                            <a href="{{ url('topcrashers:topcrashers') }}?product={{ product }}&amp;versions={{ version }}&amp;process_type=plugin&amp;days={{days}}">
```

Comment:
hmm... we have that `change_query_string` but it works based off the current query string and we don't want that, we want to start afresh.